### PR TITLE
0.9.0-rc0

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
Will look at the additional PRs submit in the last day to decide it we want to include them in this release.

**Review by dbluhm — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,116 @@
...
+[CredX from the indy-shared-rs repository]: https://github.com/hyperledger/indy-shared-rs
+[Indy SDK]: https://github.com/hyperledger/indy-sdk
+[Indy SDK to Askar migration document]: https://aca-py.org/main/deploying/IndySDKtoAskarMigration/
+[aca-py.org]: https://aca-py.org
+
+The second big change in this release is that we have upgraded the primary
+Python version from 3.6 to 3.9 (PR [\#2247]). In this case, primary means that
+Python 3.9 is used to run the unit and integration tests on all Pull Requests.
+We also do nightly runs of the main branch using Python 3.10. As
+of this release we have **dropped** Python 3.6, 3.7 and 3.7 and introduced new
```
Comment (WadeBarnes) [inline CHANGELOG.md:29]:
```suggestion
of this release we have **dropped** Python 3.6, 3.7 and 3.8 and introduced new
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Looks good other than one minor typo.

File: CHANGELOG.md (hunk 1 above)
Comment (swcurran) [reply CHANGELOG.md]: Thanks —fixed.

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**